sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/perf/Scenarios/ActivityKindServer.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Monitor.OpenTelemetry.Exporter;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+using Azure.Test.Perf;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+
+namespace Azure.Monitor.OpenTelemetry.AspNetCore.Perf
+{
+    public class ActivityKindServer : PerfTest<PerfOptions>
+    {
+        // please refer to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/perf/TemplateClientTest.cs to write perf test.
+        /* How to run
+         * Build
+            dotnet build -c Release -f <TargetFramework>
+         * Run
+         * dotnet run -c Release -f net7.0 --no-build --project <Path to this Project> ActivityKindServer --sync true
+        */
+        private const string ActivitySourceName = nameof(ActivityKindServer);
+        private readonly Batch<Activity> _activityBatch;
+        private readonly AzureMonitorTraceExporter _traceExporter;
+
+        public ActivityKindServer(PerfOptions options) : base(options)
+        {
+            Activity.DefaultIdFormat = ActivityIdFormat.W3C;
+            Activity.ForceDefaultIdFormat = true;
+
+            var listener = new ActivityListener
```

Comment:
Is there a reason to create a listener instead of relying on OpenTelemetry TracerProvider?

Reply:
Just for simplicity.